editor/code/package.json

```diff
@@ -96,7 +96,7 @@
       },
       {
         "command": "coq-lsp.document",
-        "title": "Coq LSP: Get document in serialized format, print in console"
+        "title": "Coq LSP: Get document data in serialized JSON format, display in new text editor"
```

Comment:
```suggestion
        "title": "Coq LSP: Serialize document to JSON"
```

Reply:
Actually the main verb here now is "Display"

Reply:
Reworked a bit, we can tweak further in #699